app/models/concerns/host_common.rb

```diff
@@ -204,6 +204,24 @@ def available_puppetclasses
     environment.puppetclasses - parent_classes
   end
 
+  def inherited_params(include_source = false)
+    hp = {}
+    params = host_inherited_params_objects
+    params.each do |param|
+      case param
+        when CommonParameter
+          hp.update(Hash[param.name => include_source ? {:value => param.value, :source => N_('common').to_sym, :safe_value => param.safe_value} : param.value])
+        when GroupParameter
+          hp.update(Hash[param.name => include_source ? {:value => param.value, :source => N_('hostgroup').to_sym, :safe_value => param.safe_value, :source_name => hostgroup.title} : param.value])
+        else
+          source = param.class.to_s.gsub('Parameter', '').downcase
+          source = 'operatingsystem' if source == 'os'
```

Comment:
Manipulating `source` like this won't work with `n_()`.  `n_()` is used to extract strings, so they need to be explicitly listed *somewhere* - doing `n_(source)` doesn't actually extract anything.